sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -290,6 +301,30 @@ class wi_element<sycl::ext::oneapi::bfloat16, NumRows, NumCols, Use, Layout,
 #endif // __SYCL_DEVICE_ONLY__
 };
 
+template <typename T, size_t NumRows, size_t NumCols, use Use, layout Layout,
+          typename Group>
+inline __SYCL_ALWAYS_INLINE std::tuple<uint32_t, uint32_t>
+get_coord(wi_element<T, NumRows, NumCols, Use, Layout, Group> &we) {
+#if defined(__SYCL_DEVICE_ONLY__) && defined(__SPIR__)
+  __ocl_vec_t<uint32_t, 2> coord =
+      __spirv_JointMatrixGetElementCoordINTEL(we.M.spvm, we.idx);
```

Comment:
Yes, I believe when lowering, if we cannot find a function to replace this call, a compilation error will be triggered.

Reply:
In the worst case it will be "undefined symbol error". But AFAIK now it never happens on Intel CPU and GPU as they have SPIR-V module checkers. For CPU it would say `Unsupported Capability%Name%` (get coord will have outstanding capability from joint matrix) and for GPU it will be `Unsupported Op code` or something similar.